### PR TITLE
vCon Templates

### PR DESCRIPTION
**Pull request by SteveLasker**
Adds a new Templates section to the docs, enabling a collection of industry, vertical, and partner templates.
image

### CODE REVIEW

File: content/developers/templates/vcons/index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    "subject": "vcon://xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+    "vcon_draft_version": "00"
+  }
+}
+```
+
+### arc_display_type (REQUIRED)
+
+Default within DataTrails to categorize events.
+`arc_display_type` is also the default means to [configure permissions](https://docs.datatrails.ai/platform/administration/sharing-access-inside-your-tenant/) for which type of events a client my view/edit.
```
Comment (rcoxhill-01) [inline content/developers/templates/vcons/index.md:59]: "my" -> "may"

File: content/developers/templates/vcons/index.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+### vcon_operation (REQUIRED)
+
+A DataTrails Event should be created for each completed vCon operation.
+For every creation and update to a vCon, a SCITT Statement would seal the vCon, recording it on the ledger for inclusion and verification.
+The lifecycle events of a vCon will likely evolve as a standard.
+For now, the `vcon_operation` (`string`) is the placeholder.
+
+### payload_hash_alg (REQUIRED)
+
+The hash algorithm used to has the vCon.
```
Comment (rcoxhill-01) [inline content/developers/templates/vcons/index.md:72]: "has" -> "hash"

File: content/developers/templates/vcons/index.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+### payload_preimage_content_type (REQUIRED)
+
+The property name comes from [draft-ietf-cose-hash-envelope](https://datatracker.ietf.org/doc/draft-steele-cose-hash-envelope/), representing the `content-type` of the vCon, prior to hashing.
+
+[Section 5.3.1 of vCon 00](https://www.ietf.org/archive/id/draft-vcon-vcon-container-00.html#section-5.3.1) specifies `application/vcon`.
+There is [vcon issue](https://github.com/ietf-wg-vcon/draft-ietf-vcon-vcon-container/issues/7), and discussion for using `application/vcon+json`
+
+### payload_hash_value (REQUIRED)
+
+The hash of the vCon as its recorded on DataTrails.
```
Comment (rcoxhill-01) [inline content/developers/templates/vcons/index.md:84]: "its" -> "it is" or "it's"

File: content/developers/templates/vcons/index.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+### vcon_pipeline (OPTIONAL)
+
+vCons are processed by workflow pipelines that run multiple steps.
+In the [conserver model](https://www.conserver.io/), these are called chains which run one ore more links.
```
Comment (rcoxhill-01) [inline content/developers/templates/vcons/index.md:89]: "ore" -> "or"

File: content/developers/templates/vcons/index.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+Diagnostics and tracking for the source of the request.
+Typically set to the code or service.
+For example: (`DataTrails-User-Agent:oss/conserverlink/0.1.0`)
+
+### Partner_ID (REQUIRED)
+
+Diagnostics and tracking of the Partner making requests.
+This header is independent of the [DataTrails-User-Agent](#datatrails-user-agent), as multiple services may be running the same codebase, such as the [vCon Conserver](https://github.com/vcon-dev/vcon-server/)
+
+For example: (`DataTrails-Partner-ID:synsation.io"`
```
Comment (rcoxhill-01) [inline content/developers/templates/vcons/index.md:122]: Missing ")"

File: content/developers/templates/vcons/index.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+  }
+}
+```
+
+### arc_display_type (REQUIRED)
+
+Default within DataTrails to categorize events.
+`arc_display_type` is also the default means to [configure permissions](https://docs.datatrails.ai/platform/administration/sharing-access-inside-your-tenant/) for which type of events a client my view/edit.
+
+For the purposes of permissions, this property is a duplicate of the [vcon_operation](#vcon_operation)
```
Comment (JAG-UK) [inline content/developers/templates/vcons/index.md:61]:
```suggestion
For simplicity in configuring permissions, this property is a duplicate of the [vcon_operation](#vcon_operation) but could vary in advanced scenarios.
```

File: content/developers/templates/vcons/index.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+### vcon_operation (REQUIRED)
+
+A DataTrails Event should be created for each completed vCon operation.
+For every creation and update to a vCon, a SCITT Statement would seal the vCon, recording it on the ledger for inclusion and verification.
+The lifecycle events of a vCon will likely evolve as a standard.
```
Comment (JAG-UK) [inline content/developers/templates/vcons/index.md:67]:
```suggestion
The defined lifecycle events of a vCon will likely evolve with the standard.
```

**Review by JAG-UK — CHANGES_REQUESTED** [review]

**Review by JAG-UK — APPROVED** [review]
LGTM

**Pull request merged**